src/mavlink/mavlink_camera.rs

```diff
@@ -35,6 +35,7 @@ pub struct MavlinkCameraInformation {
     mavlink_stream_type: mavlink::common::VideoStreamType,
     video_stream_uri: Url,
     video_source_type: VideoSourceType,
+    thermal: bool,
```

Comment:
This should be an optional.

Reply:
Just to document, it could, but because the ExtendedConfiguration struct implements the Default trait, we'll always have a value available for it.